CDCgov/prime-reportstream
# Oo/j gilmore a6/6422/programmers guide update

This PR updates the Programmer's guide HL7 example so that it can be copy/pasted into the validation service and pass validation. It also adds an embedded HL7 example file.

Test Steps:
1. Copy/Paste text from Sample HL7 2.5.1 Payload and Output section of Programmer's Guide into new text file with .hl7 extension.
2. Verify that new message passes validation service https://staging.reportstream.cdc.gov/file-handler/validate.
3. Repeat step 2 with embedded HL7 file.

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing
- [X ] Tested locally?

### Process

## Linked Issues
- Fixes #6422 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
